### PR TITLE
[MU4] Removed old code (in the master only keeps MU4)

### PR DESCRIPTION
**Pull request by igorkorsukov**
Removed old code (in the master only keeps MU4)

### CODE REVIEW

File: main/main.cpp [hunk 1]
```diff
@@ -18,7 +18,6 @@
 //=============================================================================
 
 #include "config.h"
-#include "mscore/musescore.h"
 #include "modulessetup.h"
 
 #ifdef BUILD_UI_MU4
```
Comment (RomanPudashkin) [inline main/main.cpp:23]: You missed this

Comment (igorkorsukov) [reply main/main.cpp:23]: I plan to do it little by little. One of the next steps will be to remove this option. Thanks

File: mu4/commonscene/exampleview.cpp [hunk 2]
```diff
@@ -11,7 +11,7 @@
 //=============================================================================
 
 #include "exampleview.h"
-#include "mscore/preferences.h"
+//#include "mscore/preferences.h"
```
Comment (RomanPudashkin) [inline mu4/commonscene/exampleview.cpp:14]: Can we just remove this header?

File: main/main.cpp (hunk 1 above)
Comment (igorkorsukov) [reply main/main.cpp:23]: "Rough strokes" are now done :)

File: mu4/commonscene/exampleview.cpp (hunk 2 above)
Comment (igorkorsukov) [reply mu4/commonscene/exampleview.cpp]: removed

**Pull request merged**